api/server.go

```diff
@@ -24,6 +26,8 @@ type ServerConfig struct {
 
 // NewServer creates a new server.
 func NewServer(conf ServerConfig) (*standard.Server, error) {
+	log.Warn(os.Getenv("PREFIX_ROUTE"))
```

Comment:
Remove the `warn` level log. If you think it's useful for debugging you can do it with `debug` level.